agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -242,6 +242,52 @@ if [ $lcl_cnt -gt 0 ]; then
 fi
 wait
 
+stockpile_path=$(getconf.py stockpile_path stockpile)
+stockpile_user=$(getconf.py stockpile_user stockpile)
+stockpile_local_remote_user=$(getconf.py local_remote_user stockpile)
+stockpile_host_remote_user=$(getconf.py host_remote_user stockpile)
+stockpile_output_path=$(getconf.py stockpile_output_path stockpile)
+stockpile_dependencies=$(getconf.py -l stockpile_dependencies packages)
+
+# install stockpile and its dependencies
+for package in $stockpile_dependencies; do
+	which $package &>/dev/null
+	if [[ $? != 0 ]]; then
+		debug_log "[$script_name] $package is not installed on the system, installing"
+		echo "$package is not installed on the system, installing"
+		yum install $package -y &>/dev/null
+	fi
+done
+if [[ -d "$stockpile_path/stockpile" ]]; then
```

Comment:
So we now have to git clone every time we need to run stockpile?
And we're putting the repo under /opt/pbench-agent?  I am not sure /opt/pbench-agent is the best place for ephemeral storage.  It would probably be more appropriate in /var/lib/pbench-agent.